app/views/claimToAdjustPoa/EnterPoAAmountView.scala.html

```diff
@@ -81,7 +81,7 @@
 }
 
 @getRow1 = @{
-    if(viewModel.poaPreviouslyAdjusted) {
+    if(viewModel.poaPreviouslyAdjusted.contains(true)) {
```

Comment:
if we can move this part of the logic on the viewModel level ?
`> viewModel.poaPreviouslyAdjusted.contains(true)`